--- cfn-lint-0.56.4/setup.py.orig	2022-01-10 17:31:50.000000000 +0100
+++ cfn-lint-0.56.4/setup.py	2023-11-19 09:30:54.858019674 +0100
@@ -53,7 +53,7 @@ that could potentially be improved'),
         'networkx~=2.4;python_version>="3.5"',
         'networkx<=2.2;python_version<"3.5"',
         'junit-xml~=1.9',
-        'pyrsistent<=0.16.0;python_version<"3.5"',
+        'pyrsistent<0.17;python_version<"3.5"',
         'jschema_to_python~=1.2.3',
         'sarif-om~=1.0.4'
     ],
